src/lib/Entry.ts

```diff
@@ -14,13 +15,31 @@ type EntryArgs = {
   del?: boolean;
   rename?: boolean;
   placeholder?: boolean;
+  uploaded?: number;
```

Comment:
S: Another minor request, but could this be `uploadedBytes` (or `uploadedSize`?) so it's clear why it's a number please?

Reply:
Thanks for this!